pandas/io/formats/info.py

```diff
@@ -87,26 +96,26 @@ class BaseInfo(ABC):
 
     Parameters
     ----------
-    data : FrameOrSeries
+    data : FrameOrSeriesUnion
         Either dataframe or series.
     memory_usage : bool or str, optional
         If "deep", introspect the data deeply by interrogating object dtypes
         for system-level memory consumption, and include it in the returned
         values.
     """
 
-    def __init__(
-        self,
-        data: FrameOrSeries,
-        memory_usage: Optional[Union[bool, str]] = None,
-    ):
-        self.data = data
-        self.memory_usage = _initialize_memory_usage(memory_usage)
+    data: FrameOrSeriesUnion
+    memory_usage: Union[bool, str]
 
     @property
-    @abstractmethod
-    def ids(self) -> Index:
-        """Column names or index names."""
+    def dtypes(self) -> Iterable[Dtype]:
+        """Dtypes.
```

Comment:
newline after triple-quote

if there's nothing useful to write, leave it blank

Reply:
Changed that.